tasks/server.coffee

```diff
@@ -28,6 +29,7 @@ module.exports = (grunt) ->
     apiProxyPrefix  = grunt.config.get("server.apiProxy.prefix") || undefined
     apiProxyHost = grunt.config.get("server.apiProxy.host") || "localhost"
     apiProxyChangeOrigin = grunt.config.get("server.apiProxy.changeOrigin")
+    apiProxyUseSsl = process.env.API_USE_SSL || grunt.config.get("server.apiProxy.useSsl") || false
```

Comment:
If you're not using it, let's nix it in the interest of just making the minimal change